lib/src/core/message/reaction.dart

```diff
@@ -28,21 +38,20 @@ class Reaction implements IReaction {
   late final IEmoji emoji;
 
   /// Creates an instance of [Reaction]
-  Reaction(RawApiMap raw) {
+  Reaction(RawApiMap raw, this.client) {
```

Comment:
if client is passed only for emoji in constructor then class property its not needed

Reply:
But, there's the `Reaction.event` method below

Reply:
I dont understand what you mean. Public property of INyxx in Reaction is not needed since its used only to pass to  `ResolvableGuildEmojiPartial` in line 49. you can make simple parameter and pass it there

Reply:
Ohh, ok, got that, sorry, I didn't understood at first